doc/Makefile

```diff
@@ -1,34 +1,39 @@
-.PHONY: all outputs clean tests clean-images
+# Makefile for Sphinx documentation
+#
 
-PYTHON ?= python
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = gcovr
+SOURCEDIR     = source
+BUILDDIR      = build
 
+PYTHON ?= python
 SCREENSHOT_FROM_HTML = wkhtmltoimage --width 700 --quality 5
+SCREENSHOTS = source/_static/screenshot-html.png source/_static/screenshot-html-details.example.cpp.png
 
-SCREENSHOTS = screenshot-html.png screenshot-html-details.example.cpp.png
-
-%.html: %.rst outputs
-	rst2html5.py --embed-stylesheet $< $@
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-all: guide.html
+.PHONY: help Makefile all outputs clean tests clean-images
 
-html: guide.html
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile $(SCREENSHOTS) outputs
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-guide.html: $(SCREENSHOTS)
-
-screenshot-%.png: examples/example-%.html
+source/_static/screenshot-%.png: examples/example-%.html
 	$(SCREENSHOT_FROM_HTML) $< $@
 
 examples/example-%.html:
 	cd examples; ./example_html.sh
 
 outputs:
-	../scripts/gcovr -h > examples/gcovr.out
+	gcovr -h > examples/gcovr.out
```

Comment:
How does gcovr end up in ../scripts? Maybe this is missing in some documentation or i did not find it.

Reply:
Thank you for spotting this problem. Originally gcovr was a single file in the `scripts` directory. Fairly recently, it was refactored into a proper module that provides a `gcovr` executable upon installation. So your change is correct.